rr_platform/src/motor_relay_node/motor_relay_node.cpp

```diff
@@ -104,7 +135,7 @@ int main(int argc, char **argv) {
     nhp.param(std::string("serial_port"), serial_port_name, std::string("/dev/ttyACM0"));
     boost::asio::io_service io_service;
     boost::asio::serial_port serial(io_service, serial_port_name);
-    serial.set_option(boost::asio::serial_port_base::baud_rate(115200));
+    serial.set_option(boost::asio::serial_port_base::baud_rate(9600));
```

Comment:
It was changed to match the arduino, where 9600 baud seemed pretty common. but we can push the baud in the firmware higher (https://arduino.stackexchange.com/questions/296/how-high-of-a-baud-rate-can-i-go-without-errors) so I am open to doing so.